extensions/amp-list/0.1/amp-list.js

```diff
@@ -60,7 +61,7 @@ export class AmpList extends AMP.BaseElement {
     /**
      * Latest fetched items to render and the promise resolver and rejecter
      * to be invoked on render success or fail, respectively.
-     * @private {?{items:!Array, resolver:!Function, rejecter:!Function}}
+     * @private {?{data:(string|!Array), resolver:!Function, rejecter:!Function}}
```

Comment:
nit: It's a good practice to make these structs into named typedefs for code clarity

Reply:
acknowledged. Leaving for now. Thanks